### PR TITLE
Feat: Add species

### PR DESCRIPTION
**Pull request by arnaudon**
Adding species entry to separate also between them (not just regions)

### CODE REVIEW

**Review by adrien-berchet — APPROVED** [review]
:partying_face:

File: synthdb/synthesis_inputs/luigi_configs/luigi_mouse_Isocortex.cfg [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+[SynthesisConfig]
+axon_method = synthesis
+
+[RunnerConfig]
+nb_jobs = 1
+
+[BuildMorphsDF]
+neurondb_path = /gpfs/bbp.cscs.ch/project/proj82/home/gevaert/morphology_release/mouse-scaled/scaled_output/06_RepairUnravel-asc/neuronDB.xml
+morphology_dirs = {"morphology_path": "/gpfs/bbp.cscs.ch/project/proj82/home/gevaert/morphology_release/mouse-scaled/scaled_output/06_RepairUnravel-asc","morphology_path_h5": "/gpfs/bbp.cscs.ch/project/proj82/home/gevaert/morphology_release/mouse-scaled/scaled_output/06_RepairUnravel-h5"}
```
Comment (adrien-berchet) [inline synthdb/synthesis_inputs/luigi_configs/luigi_mouse_Isocortex.cfg:14]: These URLs are useless, right?

File: synthdb/synthesis_inputs/luigi_configs/luigi_mouse_Isocortex.cfg [hunk 2]
```diff
@@ -0,0 +1,14 @@
+[GetSynthesisInputs]
+url = git@bbpgitlab.epfl.ch:neuromath/synthdb.git
```
Comment (adrien-berchet) [inline synthdb/synthesis_inputs/luigi_configs/luigi_mouse_Isocortex.cfg]: You can update this URL

File: synthdb/synthesis_inputs/luigi_configs/luigi_mouse_Isocortex.cfg (hunk 1 above)
Comment (arnaudon) [reply synthdb/synthesis_inputs/luigi_configs/luigi_mouse_Isocortex.cfg:14]: I keep them for now, then they'll be on AWS, but never public, this code is not really to be use as public code in fact

File: synthdb/synthesis_inputs/luigi_configs/luigi_mouse_Isocortex.cfg (hunk 2 above)
Comment (arnaudon) [reply synthdb/synthesis_inputs/luigi_configs/luigi_mouse_Isocortex.cfg]: this one I can!

**Pull request merged**